gateway/utils/validate_test.go

```diff
@@ -356,6 +356,38 @@ func TestValidate(t *testing.T) {
 			},
 			want: true,
 		},
+		{
+			name: "valid contains",
+			args: args{
+				where: map[string]interface{}{"op2": map[string]interface{}{"$contains": map[string]interface{}{"a": 1, "b": "one", "c": true,}}},
+				obj:   map[string]interface{}{"op2": map[string]interface{}{"a": 1, "b": "one", "c": true,}},
+			},
+			want: true,
+		},
+		{
+			name: "invalid contains",
```

Comment:
Add one more test case, what if some keys match and some don't

Reply:
not required

Reply:
Add a test case. Whats wrong in that